### PR TITLE
Redesign UI

### PR DESCRIPTION
**Pull request by alchuu00**
Refactored website from multi-page to single-page layout for improved user experience and navigation.

### CODE REVIEW

File: frontend/app/page.tsx [hunk 1]
```diff
@@ -79,47 +96,145 @@ export default function Home() {
...
+            Worried about privacy? We have got you covered. Your data is
+            precious, and we treat it that way. No need to fret about your
+            information lingering in the cloud. Your uploaded resume is promptly
+            removed after processing. Plus, you have control the JSON data file,
+            holding extracted info from your resume, can be wiped manually at
+            your request or automatically after a few hours.
+          </p>
+          <p className="p-2">
+            Still have privacy concerns? Check out our app repository on
+            <a href="https://github.com/PooriaT/Resumivise">GitHub</a>
```
Comment (PooriaT) [inline frontend/app/page.tsx:181]: Here, a space is required before and after GitHub.

File: frontend/app/page.tsx [hunk 2]
```diff
@@ -79,47 +96,145 @@ export default function Home() {
...
+          <p className="p-2 text-red-500">
+            Disclaimer: While Resumivise is here to assist, please note that we
+            do not take responsibility for any issues that may arise in your
+            resume during the application process. It is crucial to review your
+            resume thoroughly before hitting that apply button.
+          </p>
+          <p className="p-2">
+            Ready to take the next step in your career? It is time to apply with
+            confidence, thanks to Resumivise!
+          </p>
```
Comment (PooriaT) [inline frontend/app/page.tsx:193]: At the bottom of the About section, I would like to add a part related to team members which refers to my LinkedIn profile and yours—something like a thumbnail of our LinkedIn profiles.  Also, please add the Contact Us part, the same as what is available on the live version of the application. Only provide the email address.

**Comment by alchuu00** [comment]
Okay, thank you @PooriaT for the comments. I will revise.

**Comment by PooriaT** [comment]
> Okay, thank you @PooriaT for the comments. I will revise.

@alchuu00 

I provided you with some more information on Discord.

**Pull request merged**